### PR TITLE
SEO-184192 Xamarin Page with Redirect

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan ,
Have modified title, H1 and description, please review.
Regards,
Yvone.

### CODE REVIEW

File: Xamarin/AutoComplete/Overview.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-description: This section will explain about Overview and key feaures available in Syncfusion Autocomplete in Xamarin.Forms 
+title: Overview of  Xamarin.Forms AutoComplete Control | Syncfusion
+description: Learn here all about overview of Syncfusion Xamarin.Forms AutoComplete control, its elements, and more.
 platform: xamarin
 control: SfAutoComplete
 documentation: ug
 ---
 
-# Xamarin AutoComplete (SfAutoComplete) Overview
+#  Overview of Xamarin AutoComplete (SfAutoComplete)
```
Comment (gowthamanthavasiyappan) [inline Xamarin/AutoComplete/Overview.md:10]: Remove extra space here

**Review by gowthamanthavasiyappan — COMMENTED** [review]
Commented

File: Xamarin/AutoComplete/Overview.md (hunk 1 above)
Comment (Yvone-Atieno) [reply Xamarin/AutoComplete/Overview.md]: removed

Comment (Yvone-Atieno) [reply Xamarin/AutoComplete/Overview.md]: removed

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**